tests/make/make.spec.js

```diff
@@ -220,6 +220,36 @@ describe('docker-compose.yml', () => {
         }
       }
     });
+
+    describe.each(['development', 'production'])(
+      'DOCKER_TARGET=%s',
+      (DOCKER_TARGET) => {
+        const FILTERED_KEYS = [
+          'DOCKER_COMMIT',
+          'DOCKER_VERSION',
+          'DOCKER_BUILD',
+        ];
+        it('.services.(web|worker).environment excludes build info variables', () => {
```

Comment:
I forgot to add this last time: I'm not clear what this is testing, exactly.  What aren't we including in the web/worker envs, and why?

Reply:
We exclude variables that could be interpreted as build info such as the DOCKER_COMMIT, DOCKER_VERSION and (eventually) DOCKER_TARGET. You don't want to live in a world where you build with one commit/version and then run the container with another value set (arbitrarily) and then your container thinks it is a different image.. This test confirms that the variables that are used in the build info file are NOT available on the environment of the container.